app/models/ui/ConfirmationPanel.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.ui
+
+case class ConfirmationPanel (text: Text, additionalText: Seq[Text]) extends UIComponent
```

Comment:
this could be a single Seq[Text], to allow for one or more lines

Reply:
As ConfirmationPanel is a subclass of UIComponent it inherits "text". As the panel has a main title followed by additional lines of text it seemed reasonable to implement the class with "text" and "additionalText".

Reply:
ok